Description: reconfigure CMakeList.txt defaults
 Defaults changed to fix build failure.
 C++11 isn't enough, GNU++11 standard is required.
 Build without -Werror because of a left shift operation on a signed number.
 .
 pulseview (0.3.0-2) unstable; urgency=medium
 .
   * Grab maintainership to avoid NMU warning
   * Update control borrowing heavily from Daniel Elstner's PPA
   * Ubuntu to Debian package name issue
   * Add qtbase5-dev dependency
   * Disable -Werror because of shifting a signed number upstream
   * Change to gnu++11 C++ standard
Author: Aaron Brady <aaron@debian9.insom.me.uk>

--- pulseview-0.3.0.orig/CMakeLists.txt
+++ pulseview-0.3.0/CMakeLists.txt
@@ -30,7 +30,7 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_S
 #= User Options
 #-------------------------------------------------------------------------------
 
-option(DISABLE_WERROR "Build without -Werror" FALSE)
+option(DISABLE_WERROR "Build without -Werror" TRUE)
 option(ENABLE_SIGNALS "Build with UNIX signals" TRUE)
 option(ENABLE_DECODE "Build with libsigrokdecode" TRUE)
 option(ENABLE_TESTS "Enable unit tests" TRUE)
@@ -345,7 +345,7 @@ endif()
 add_definitions(${QT_DEFINITIONS} -DQT_NO_KEYWORDS)
 add_definitions(-D__STDC_LIMIT_MACROS)
 add_definitions(-Wall -Wextra)
-add_definitions(-std=c++11)
+add_definitions(-std=gnu++11)
 
 if(ENABLE_DECODE)
 	add_definitions(-DENABLE_DECODE)
